### DISCUSSION TITLE
how can I create a page like typography

### DISCUSSION THREAD

**Discussion by Shentia** [UX-and-I/storybook-design-token#182] [Q&A] [answered]
I want to have a page like this demo. I installed addon and works in tabs but I want to have a page like demo page 
for typography
 
https://storybook-design-token-v1.netlify.app/?path=/story/design-tokens-typography--page
Thanks

**Comment by Shentia** [comment]
I found it from here 👍🏼 

https://github.com/UX-and-I/storybook-design-token/blob/v1/demo/src/design-tokens/typography.stories.mdx

**Comment by Sqrrl** [comment, marked as answer]
Also documented here: https://github.com/UX-and-I/storybook-design-token#design-token-doc-block
